backend/runner/rollbackrun/runner.go

```diff
@@ -101,11 +102,18 @@ func (r *Runner) generateRollbackSQL(ctx context.Context, task *store.TaskMessag
 	var rollbackSQLStatus api.RollbackSQLStatus
 	var rollbackStatement, rollbackError string
 
-	rollbackSQL, err := r.generateRollbackSQLImpl(ctx, task, payload)
+	binlogSizeLimit := 8 * 1024 * 1024
+	rollbackSQL, err := r.generateRollbackSQLImpl(ctx, task, payload, binlogSizeLimit)
 	if err != nil {
 		log.Error("Failed to generate rollback SQL statement", zap.Error(err))
 		rollbackSQLStatus = api.RollbackSQLStatusFailed
-		rollbackError = err.Error()
+		if strings.Contains(err.Error(), "failed to parse binlog extension") {
```

Comment:
Not a big fan of doing this. Do we have other ways?

Reply:
updated